src/Orleans/Streams/Predicates/RegexStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    public class RegexStreamNamespacePredicate : IStreamNamespacePredicate
+    {
```

Comment:
Actually I found 0 warnings. But I think it still make sense to add some xml comment on the new predicate and RegexImplicitStreamSubscriptionAttribute . But other than that. I Approve!

Thanks for the work ! It is great